### PR TITLE
Feature/update documentation

### PR DESCRIPTION
**Pull request by perruche**
**Add documentation about the post installation process**
- in the shell after the project is created
- in the readme of this project

**Add the `WP_DEFAULT_THEME` constant to automatically activate our theme**

**Add the execution of `bin/generate-wp-config.sh`script to the project creation process**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,12 +8,25 @@
...
 ## Usage
 
+### Install
 Run the following command to bootstrap a WordPress project using Studio Meta's tools and workflows:
 
 ```
 npx @studiometa/create-wordpress-project project-name
 ```
 
+Follow the prompt steps to complete the project creation
```
Comment (titouanmathis) [inline readme.md]: There is a `.` missing at the end of the sentence

File: saofile.js [hunk 2]
```diff
@@ -136,22 +144,28 @@ module.exports = {
...
     console.log(chalk`${tab}{cyan bin/get-wp-salts.sh}\n`);
     console.log(chalk`${tab}Install the composer dependencies:`);
     console.log(chalk`${tab}{cyan composer install}\n`);
+    console.log(chalk`${tab}Create Database (using info from .env):`);
+    console.log(chalk`${tab}{cyan ./vendor/bin/wp db create}\n`);
+    console.log(chalk`${tab}Install WordPress:`);
+    console.log(
+      chalk`${tab}{cyan ./vendor/bin/wp core install --url="${this.answers.url}" --admin_user="<ADMIN_USER>" --admin_email="<ADMIN_EMAIL>" --title="<SITE_TITLE>"}\n`
+    );
+    console.log(chalk`${tab}Install development dependancies:`);
```
Comment (titouanmathis) [inline saofile.js]: There is a typo in `dependencies`.

**Review by titouanmathis — CHANGES_REQUESTED** [review]
Some minor typo to fix, all good for me otherwise!

**Review by titouanmathis — APPROVED** [review]
👍

**Review by lusimeon — APPROVED** [review]

**Review by titouanmathis — APPROVED** [review]

**Pull request merged**